### PR TITLE
Working on hud branch

### PR DESCRIPTION
**Pull request by augnor**
I have created an almost working hud, (no updating functions yet for hp, xp and score) but otherwise working. resolved merge conflicts when I fetched main before

### CODE REVIEW

File: core/src/View/HUD.java [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+import com.badlogic.gdx.scenes.scene2d.ui.Stack;
+import com.badlogic.gdx.scenes.scene2d.ui.Table;
+import com.badlogic.gdx.utils.viewport.FitViewport;
+
+public class HUD {
+    private Stage stage;
+    private FitViewport stageViewport;
+
+    public HUD(SpriteBatch spriteBatch) {
+        stageViewport = new FitViewport(800,480);
```
Comment (pkasinski) [inline core/src/View/HUD.java]: The code could be separated into separate methods to make it more readable.

File: core/src/View/HUD.java [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+        stage = new Stage(stageViewport, spriteBatch);
+
+        int score = 10;
```
Comment (pkasinski) [inline core/src/View/HUD.java]: Why is the score 10?

File: core/src/View/HUD.java [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+
+        Texture xpyellow = new Texture(Gdx.files.internal("HudColors/Xpyellowc8a72b.png"));
+        NinePatch xpyellow2 = new NinePatch(xpyellow, 0, 0, 0, 0);
+        Image fullxpbar = new Image(xpyellow2);
+
+        Label hpLabel = new Label("HP", new Label.LabelStyle((new BitmapFont()), Color.WHITE));
+        Label scoreLabel = new Label("Score: " + Integer.toString(score), new Label.LabelStyle((new BitmapFont()), Color.WHITE));
+
+        //3 tables. Id like to have one but stack is not working like I think it should. Currently
+        // it acts as 3 layers, black under, secondary color, main color, text? as a fourth if I want
```
Comment (pkasinski) [inline core/src/View/HUD.java:60]: If the tables act as three different layers then maybe they could have better names than table1, table2 and table3.

**Review by rojtekin — APPROVED** [review]
LGTM, just needs functional decomposition

**Review by pkasinski — COMMENTED** [review]
I left a few comments. I think the code should be separated into smaller parts.

**Review by pkasinski — APPROVED** [review]

**Pull request merged**